Doc/library/functions.rst

```diff
@@ -506,6 +506,9 @@ are always available.  They are listed here in alphabetical order.
    returns the current global and local dictionary, respectively, which may be
    useful to pass around for use by :func:`eval` or :func:`exec`.
 
+   If the given source is a string, then the leading whitespace will be
+   omitted.
```

Comment:
```suggestion
   If the given source is a string, then all leading whitespace characters will be
   omitted.
```

Reply:
"... then leading and trailing whitespace is stripped."